proof/crefine/ARM/IsolatedThreadAction.thy

```diff
@@ -912,7 +912,7 @@ lemma setThreadState_no_sch_change:
   "\<lbrace>\<lambda>s. P (ksSchedulerAction s) \<and> (runnable' st \<or> t \<noteq> ksCurThread s)\<rbrace>
       setThreadState st t
    \<lbrace>\<lambda>rv s. P (ksSchedulerAction s)\<rbrace>"
-  (is "NonDetMonad.valid ?P ?f ?Q")
+  (is "NonDetMonadVCG.valid ?P ?f ?Q")
```

Comment:
While I do understand, it is a bit sad we have to now type a longer prefix, because we often use find_theorems with `-NondetMonad.valid`.